docs/chiadatalayer.md

```diff
@@ -105,33 +105,38 @@ spec:
 
 If you use a hostPath volume in a kubernetes cluster with multiple nodes, make sure you have set the proper nodeSelector to ensure the data_layer Pod only runs on the node with that hostPath.
 
-In all three cases, the volume will be automatically mounted in the chia container at `/datalayer/server`. You can change the mount path for the server files in the fileserver container though, see the `Fileserver Configuration` section for information on that.
+In all three cases, the volume will be automatically mounted in the chia container at `/datalayer/server`.
 
 See the [Storage](storage.md) documentation for information on CHIA_ROOT persistence, which is separate from the server files volume.
 
 ## Fileserver Configuration
 
-The ChiaDataLayer can optionally run a fileserver sidecar container to serve the data_layer server files. This is disabled by default but can be enabled with the following configuration:
+The ChiaDataLayer can optionally run a fileserver sidecar container to serve the data_layer server files. This will be needed if you plan to mirror data or publish your own data on DataLayer. This is disabled by default but can be enabled with these fileserver configuration options:
 
 ```yaml
 spec:
   fileserver:
     enabled: true
-    # Optional: defaults to the official chia image using data_layer_http. 
-    # But a custom image for the fileserver can be specified here.
+    # Optional: defaults to the official chia image using data_layer_http,
+    # but a custom image for the fileserver can be specified here.
     image: "custom/fileserver:tag"
-    # Optional custom mount path for server files
-    # This is the mount path in the fileserver container only. 
+    # Optional: custom mount path for server files (.dat files)
+    # This is the mount path in the fileserver container only.
     # It will not change the mount path in the chia container.
+    # This path is what webserver software would set the webroot to
     serverFileMountpath: "/custom/path"
-    # Optional custom container port
+    # Optional: custom container port. This port will be
+    # exposed on at a service endpoint on port 80 by default
+    # unless overridden below in the "service" section
     containerPort: 8080
-    # Optional service configuration
+    # Optional: service configuration. Most people will not need
+    # to change this
     service:
       enabled: true
       type: ClusterIP
       externalTrafficPolicy: Local
 ```
+ References for service endpoint configuration fields can be found in the [kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) and for the [Chia operator specifically](https://github.com/Chia-Network/chia-operator/blob/main/docs/services-networking.md). By default the .dat files will be available at a service endpoint on port 80.
```

Comment:
```suggestion

References for service endpoint configuration fields can be found in the [kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) and for the [Chia operator specifically](https://github.com/Chia-Network/chia-operator/blob/main/docs/services-networking.md). By default the .dat files will be available at a service endpoint on port 80.
```
Markdown lint is going to be mad about not having a padding line here. And also starting a line with a space.